src/popup/Popup.tsx

```diff
@@ -15,9 +15,25 @@ export default function Popup(): JSX.Element {
     chrome.runtime.sendMessage({ popupMounted: true });
   };
 
+  const [isEnabled, setIsEnabled] = useState(true);
+
+  const changeAutolike = useCallback(() => {
+    setIsEnabled(!isEnabled);
```

Comment:
```suggestion
    setIsEnabled((enabled) => !enabled);
```
removes the use of `isEnabled` which allows us to remove it from the dependency array on line 25